app/controllers/idv/document_capture_controller.rb

```diff
@@ -5,7 +5,7 @@ class DocumentCaptureController < ApplicationController
     include IdvStepConcern
     include StepIndicatorConcern
 
-    before_action :confirm_not_rate_limited
+    before_action :confirm_not_rate_limited, except: [:update]
```

Comment:
Are we still rate-limiting update somewhere? Otherwise, is it possible for someone to post unlimited updates?

Reply:
Looks like the specs you linked cover the update case. Do we want to add a line that checks that the rate limiter is limited after that final successful attempt [here](https://github.com/18F/identity-idp/blob/1b1ed172b1ad5e65b379470d6f23f4bef4697baf/spec/features/idv/doc_auth/document_capture_spec.rb#L107)?

Reply:
So, the update case for doc auth is interesting because it does not actually submit any data. It checks for the doc auth result and advances if it exist. It does not result in a vendor call. That is done by `Idv::ImageUploadsController` within `Idv::ApiImageUploadForm`. We do not need to consider the rate limiter in the update action.

Reply:
The above said, I think a line in the feature spec is reasonable and not too expensive. I'll add one.